scopes/generator/generator/generator.main.runtime.ts

```diff
@@ -199,11 +207,17 @@ export class GeneratorMain {
       throw new BitError("error: unable generating a new workspace, you're in a workspace already");
```

Comment:
`unable generating` --> `unable to generate`

Reply:
It would be even better with: `Error: unable to generate a new workspace inside of an existing workspace`